Makefile

```diff
@@ -28,6 +28,24 @@ black-lint:
 black-format:
 	black ${PROJECT_DIR} setup.py
 
+build: ## Build the project ready for deployment to pypi
```

Comment:
Maybe check https://github.com/developersociety/django-postgres-lock/blob/master/Makefile for a better version at some point?

Next time round, this is fine for now!